spring-integration-core/src/test/java/org/springframework/integration/dsl/flows/IntegrationFlowTests.java

```diff
@@ -700,7 +701,15 @@ public IntegrationFlow wireTapFlow4() {
 		public IntegrationFlow wireTapFlow5() {
 			return f -> f
 					.wireTap(sf -> sf
-							.<String, String>transform(String::toUpperCase)
+							.transform(// Must not be lambda for SpEL fallback behavior on empty payload
```

Comment:
I am not sure why the payload resolver doesn't allow an empty string but I suppose it's rare so falling back to SpEL is ok.

Reply:
I treat this like a note and not as some confusion or advice to change something.
Correct?

Reply:
Yes, it's fine; I was just making a comment; since the problem only arises with an empty payload.

Reply:
Well, it also is there when we use a `@UseSpelInvoker`.

I believe I had to come up with the `@Transformer` against that `GenericTransformer` impl test-case, but it doesn't matter so far: I don't think it makes sense to use  `@Transformer` and don't rely on powerful POJO method call support.